docs/remote-runner-introduction.md

```diff
@@ -173,6 +173,58 @@ the default snapshot stays up to date.
 For more technical details on our VM implementation, see our BazelCon
 talk [Reusing Bazel's Analysis Cache by Cloning Micro-VMs](https://www.youtube.com/watch?v=YycEXBlv7ZA).
 
+#### Remote snapshot save policy
+
+By default, after every remote run, a snapshot is cached locally on the machine
+that ran it. Subsequent runs that are assigned to the same executor can resume
+from that snapshot. Our scheduler uses affinity routing to prioritize routing
+similar workloads to the same executor, to increase the likelihood of hitting
+a local snapshot match.
+
+Snapshots can also be cached in the remote cache. There are pros and cons to
+using remote snapshots.
+
+Local snapshots cannot be guaranteed, because the executor
+that has it cached locally may be unavailable. For example if it's fully occupied
+with other workloads or has been restarted during a release, subsequent remote
```

Comment:
nit: s/has been restarted during a release/is restarting during a release/g

(since on-disk cache sticks around, it's only if the machine is down or preempted that the files are lost)